.github/workflows/deny.yml

```diff
@@ -0,0 +1,65 @@
+name: Deny - Linux
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  schedule:
+    # At midnight UTC
+    - cron: '0 0 * * *'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏left a similar comment on the Workflow Checks PR- I suspect this should probably follow the same schedule as the Component Features which only runs Tue-Sat. WDYT?

```suggestion
    # At midnight UTC
    - cron: '0 0 * * 2-6'
```

Reply:
My thought is that, unlike the component feature check which would only start failing due to a code change, this workflow will start failing if a vulnerability is reported, which could happen at any time.